packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -379,6 +405,39 @@ export default {
             }
         },
 
+        /**
+         * Redirect to the payment page.
+         */
+        redirectToPayment () {
+            setTimeout(() => { // TODO: remove this when the order team handles this automatically.
+                window.location.assign(this.paymentPageUrl);
```

Comment:
paymentPageUrl is in format /pay/[orderId]
It needs to be amended at runtime based on response from place order call.

Reply:
Good point, shouldn't have missed that. One of the problems of not being able to test everything end-to-end at the moment.